### DISCUSSION TITLE
Tree shaking and react-icons

### DISCUSSION THREAD

**Discussion by mkalafior** [themesberg/flowbite-react#1439] [Q&A]
Does anyone here know how to handle a large bundle size, mainly caused by react-icons? I want to use Flowbite to build small components for some HTML websites, but it seems that Webpack can't extract only the Flowbite components that are actually used (mostly buttons and form elements). Instead, it's including almost everything (the whole Flowbite library and entire sets of MD/FA icons, which make up about 75% of the bundle size).

**Comment by SutuSebastian** [comment]
Fixed by [flowbite-react@0.11.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.11.0) release.
